src/software/simulation/threaded_standalone_simulator.cpp

```diff
@@ -0,0 +1,193 @@
+#include "software/simulation/threaded_standalone_simulator.h"
+
```

Comment:
What was the logic behind merging `StandaloneSimulator` and `ThreadedSimulator`? It seems like we've just ended up with a class with more responsibilities?

Reply:
The idea is that since ThreadedSimulator isn't used by itself, we have extra wrapper classes that we don't need. I think that was the idea @MathewMacDougall was going for with the GUI cleanup PR

Reply:
I see what you mean, but this was definitely a bit more then a wrapper class, it's performing a fundamentally different job. I thinking merging these violates the single responsibility principle.

Reply:
Yeah I'll change this PR and we can revisit this later

Reply:
fixed